test/bitarray.jl

```diff
@@ -203,6 +203,15 @@ timesofar("utils")
         @test_throws DimensionMismatch BitMatrix((isodd(i) for i in 1:3))
     end
 
+    @testset "constructor from infinite iterable" begin
+        struct InfiniteIter; state::Bool; end
```

Comment:
```suggestion
        struct InfiniteIter end
```
The struct doesn’t need a field, it is never used.